src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ImplicitNamespaceImports.targets

```diff
@@ -0,0 +1,22 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.ImplicitNamespaceImports.targets
+
+This file contains @(Import) items for C# 10 global usings.
+
+***********************************************************************************************
+-->
+<Project>
+  <!--
+    There's a master switch, $(DisableImplicitNamespaceImports),
+    which, if set, will prevent all implicit namespace imports.
+    This is done automatically, we don't have to check
+    $(DisableImplicitNamespaceImports) here.
+  -->
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_Android)' != 'true' ">
```

Comment:
Is there *also* a `$(DisableImplicitNamespaceImports)` property?  If so, should this group be conditional on `'$(DisableImplicitNamespaceImports)' != 'true'` as well?  Or will `Microsoft.Android.Sdk.ImplicitNamespaceImports.targets` not be `<Import/>`ed when `'$(DisableImplicitNamespaceImports)' != 'true'`?

Reply:
The target that uses the `@(Import)` item group doesn't run if you set `$(DisableImplicitNamespaceImports)`:

https://github.com/dotnet/sdk/blob/93b432bd8e73bb68aa4621db8a967d62e6f0cc23/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateImplicitNamespaceImports.targets#L35-L38

Here is the commit that implements this:

https://github.com/dotnet/sdk/commit/85943958e5a01d6bbd39a947aec2a4a37b8ba7cf